<ui version="4.0" >
 <class>fileRequestWindowClass</class>
 <widget class="QWidget" name="fileRequestWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>232</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>468</width>
    <height>232</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>232</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>File request</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../qutim.qrc" >
    <normaloff>:/icons/crystal_project/save_all.png</normaloff>:/icons/crystal_project/save_all.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="sizeConstraint" >
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="leftMargin" >
    <number>6</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>6</number>
   </property>
   <property name="bottomMargin" >
    <number>7</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="fileLabel" >
     <property name="minimumSize" >
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../../../qutim.qrc" >:/icons/crystal_project/filerequest.png</pixmap>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="text" >
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fromLabel" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="text" >
        <string>IP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ipLabel" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="text" >
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fileNameLabel" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="text" >
        <string>File size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fileSizeLabel" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="acceptButton" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>Accept</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/apply.png</normaloff>:/icons/crystal_project/apply.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="declineButton" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>Decline</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/cancel.png</normaloff>:/icons/crystal_project/cancel.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../../../qutim.qrc" />
 </resources>
 <connections/>
</ui>
